recipes/simde/all/conanfile.py

```diff
@@ -0,0 +1,48 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, copy
+from conan.tools.layout import basic_layout
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class SIMEeConan(ConanFile):
+    name = "simde"
+    description = "Implementations of SIMD instruction sets for systems which don't natively support them."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/simd-everywhere/simde"
+    topics = ("neon", "avx", "sse", "simd", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def package_id(self):
+        self.info.clear()
+
+    def validate(self):
+        if self.settings.arch not in ["x86", "x86_64", "armv8"] or self.settings.os == "Emscripten":
+            raise ConanInvalidConfiguration(f"{self.ref} supports x86, x86_64, armv8 only or emscriptien.")
```

Comment:
? SIMDe supports basically every architecture

Reply:
TIL! Great to know :)

If you have more details on a proper validate method, we would appreciate the extra info/a PR fixing the issue (But you don't have to! We can do it ourselves too)